### PR TITLE
Add support for DD_PROFILING_ENDPOINT_COLLECTION_ENABLED

### PR DESCRIPTION
**Pull request by mar-kolya**
To disable endpoint collection. Enabled by default.

## Checklist
- [ ] Added to the correct milestone.
- [ ] Tests provided or description of manual testing performed is included in the code or PR.
- [ ] Library documentation is updated.
- [ ] [Corp site](https://github.com/DataDog/documentation/) documentation is updated (link to the PR).

### CODE REVIEW

File: ddtrace/profiling/event.py [hunk 1]
```diff
@@ -53,10 +53,12 @@ class StackBasedEvent(SampleEvent):
     def set_trace_info(
         self,
         span,  # type: typing.Optional[ddspan.Span]
+        endpoint_collection_enabled,
```
Comment (brettlangdon) [inline ddtrace/profiling/event.py:56]:
```suggestion
        endpoint_collection_enabled,  # type: bool
```

**Review by brettlangdon — COMMENTED** [review]
type hint comment, but otherwise lgtm

File: releasenotes/notes/profiling-option-to-disable-endpoint-collection-48ab251acb0d9b4e.yaml [hunk 2]
```diff
@@ -0,0 +1,5 @@
+---
+features:
+  - |
+    Add support for `DD_PROFILING_ENDPOINT_COLLECTION_ENABLED` env vaireble
```
Comment (P403n1x87) [inline releasenotes/notes/profiling-option-to-disable-endpoint-collection-48ab251acb0d9b4e.yaml:4]:
```suggestion
    Add support for `DD_PROFILING_ENDPOINT_COLLECTION_ENABLED` env variable
```

Comment (brettlangdon) [reply releasenotes/notes/profiling-option-to-disable-endpoint-collection-48ab251acb0d9b4e.yaml:4]:
```suggestion
    Add support for ``DD_PROFILING_ENDPOINT_COLLECTION_ENABLED`` env variable
```

I missed it, also needs double <code>``</code>` for inline code blocks.

**Review by brettlangdon — DISMISSED** [review]

**Review by brettlangdon — APPROVED** [review]

**Review by Kyle-Verhoog — APPROVED** [review]

**Pull request merged**